frontend/wallet/src/render/views/settings/DelegationSettings.re

```diff
@@ -22,9 +22,13 @@ type feeSelection =
   | DefaultAmount
   | Custom(string);
 
+type errorOutcome =
+  | InvalidDelegateKey;
+
 type modalState = {
   delegate: option(PublicKey.t),
   fee: feeSelection,
+  error: option(errorOutcome),
```

Comment:
I could imagine this might change to become an error(string) in the near future when we handle displaying graphql server errors but seems ok for now.